lib/link.js

```diff
@@ -54,27 +66,33 @@ export default class Link extends Component {
   }
 
   render () {
+    // If no children are provided (<Link></Link>). We don't render. The proptype isRequired check will only show a warning
+    if (typeof this.props.children === 'undefined') {
+      return null
+    }
+
+    // Deprecated. Warning shown by propType check. If the childen provided is a string (<Link>example</Link>) we wrap it in an <a> tag
+    if (typeof this.props.children === 'string') {
+      return <a onClick={this.linkClicked} href={this.props.as || this.props.href}>{this.props.children}</a>
+    }
+
+    // Add onClick to children
     const children = Children.map(this.props.children, (child) => {
```

Comment:
Excellent. Moving to `Children.only` allowed for the removal of the undefined / multiple children check. 👍